pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -1,34 +1,159 @@
 """
```

Comment:
The main changes, both here and in `test_timedelta.py`:

- Use fixtures to extract, simplify repeated test setup
- Split up tests with many `assert`s, and/or distinct sections of setup code, into separate tests.
- Parameterize tests, wherever sensible, over all arithmetic ops of a similar kind, e.g. `add` and `radd`
- Parameterize tests to run with `Array/Index/Series/DataFrame`s.
- Consistent grouping of like tests into classes
- More descriptive test names